apax/config/train_config.py

```diff
@@ -76,10 +76,14 @@ class PBPDatset(DatasetConfig, extra="forbid"):
     ----------
     num_workers : int
         | Number of batches to be processed in parallel.
+    reset_every : int
+        | Number of epochs before reinitializing the ProcessPoolExcecutor.
+        | Avoids memory leaks.
     """
 
     processing: Literal["pbp"] = "pbp"
     num_workers: PositiveInt = 10
+    reset_every: PositiveInt = 10
```

Comment:
Can this be set via `dataset:reset_every` or is this really hard coded? Do we know the performance trade-off for setting this larger / smaller?

Reply:
it's the config class, this allows you to set it under `dataset:processing:reset_every`